cmd/kubectl-gadget/profile/block-io.go

```diff
@@ -76,14 +137,20 @@ func runBiolatency(cmd *cobra.Command, args []string) error {
 			time.Sleep(time.Duration(params.Timeout) * time.Second)
 			c <- os.Interrupt
 		}()
-		fmt.Printf("Tracing block device I/O...")
+		if params.OutputMode != utils.OutputModeJSON {
+			fmt.Printf("Tracing block device I/O...\n")
+		}
 	} else {
 		fmt.Printf("Tracing block device I/O... Hit Ctrl-C to end.")
 	}
```

Comment:
I think this should be printed when output mode is not json, right? Otherwise a tool parsing this could fail.

Reply:
I think everybody knows that the standard way to stop a running command is Ctrl + C, so I think it's not a problem.